ros/api/v0/hosts.py

```diff
@@ -13,6 +13,9 @@
 DEFAULT_HOSTS_PER_REP = 10
 DEFAULT_OFFSET = 0
 
+SYSTEM_STATES = {"INSTANCE_OVERSIZED": "Oversized", "INSTANCE_UNDERSIZED": "Undersized",
+                 "CONSUMPTION_MODEL": "Idling", "STORAGE_RIGHTSIZING": "Storage rightsizing"}
```

Comment:
If we are to store the `state` in `systems` table, shouldn't this be defined in the processor directly?

Reply:
Currently, the state is not stored in a separate column, we are storing the JSON object in the rule_hit_details column as is.
My thought was not to store modified JSON data(rule hit) in DB but to modify it at the time of fetching through API.